build/x86_64_debian/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM debian:7
+
+#RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+RUN /bin/bash -l -c "apt-get update -y && apt-get install -y \
```

Comment:
Why are we running this inside of /bin/bash? Same comment everywhere else.

Reply:
apt needs a login shell to get rvm to work correctly. This isn't an issue for yum on the other hand.

Reply:
Actually nvm my bad. We don't need a login shell to install the dependencies. Fixed.